doc/graph/fusion_patterns/sdpa.md

```diff
@@ -123,19 +167,22 @@ platforms follow the general description in @ref dev_guide_data_types.
    shape or the shapes can be properly broadcasted based on the operation
    attribute.
 3. CPU
-   - Optimized implementation is available for 4D Q/K tensors with shape defined
-     as (N, H, S, D_qk) and V tensor with shape defined as (N, H, S, D_v).
-   - Optimized implementation is available for OpenMP runtime and Threadpool
+   - Optimized implementation for inference is available for 4D Q/K tensors with
+     shape defined as (N, H, S, D_qk) and V tensor with shape defined as
+     (N, H, S, D_v).
+   - Optimized implementation for inference is available for OpenMP runtime and Threadpool
      runtime on Intel Architecture Processors.
    - Specifically for OpenMP runtime, the optimized implementation requires `N *
      H > 2 * thread number` to get enough parallelism.
+   - Optimized implementation for training forward and backward is not available.
```

Comment:
I feel it's not needed to mention that. Let's only state what are optimized. For others, the first bullet about primitive-based implementation covers that.